### PR TITLE
Add a setting to turn on/off custom character-defined chatboxes

### PR DESCRIPTION
**Pull request by Crystalwarrior**
[ye](https://cdn.discordapp.com/attachments/363831549264789505/723622978411298826/Yotcq2G2sG.mp4
)

### CODE REVIEW

**Comment by oldmud0** [comment]
what?

i thought this was already a thing?

**Comment by Crystalwarrior** [comment]
yeah all my users were complaining it wasn't lol, currently you had to manually go into every char.ini used and remove the line that defines the chatbox.

File: src/courtroom.cpp [hunk 1]
```diff
@@ -2692,8 +2697,11 @@ void Courtroom::chat_tick()
       ui_vp_player_char->play_idle(m_chatmessage[CHAR_NAME],
                                    m_chatmessage[EMOTE]);
     }
-    QString f_char = m_chatmessage[CHAR_NAME];
-    QString f_custom_theme = ao_app->get_chat(f_char);
+    QString f_char;
+    QString f_custom_theme;
+    if (ao_app->is_customchat_enabled())
+      f_char = m_chatmessage[CHAR_NAME];
+      f_custom_theme = ao_app->get_chat(f_char);
```
Comment (oldmud0) [inline src/courtroom.cpp:2704]: hey what is this? then block with no braces?

Comment (Crystalwarrior) [reply src/courtroom.cpp:2704]: oh fuck

**Comment by oldmud0** [comment]
okay i18n is going to be super messed up after all these new features

**Pull request merged**